src/buffer.rs

```diff
@@ -888,7 +888,7 @@ where
     Container: Deref<Target = [P::Subpixel]> + DerefMut,
 {
     // TODO: choose name under which to expose.
-    fn inner_pixels_mut(&mut self) -> &mut [P::Subpixel] {
+    pub(crate) fn inner_pixels_mut(&mut self) -> &mut [P::Subpixel] {
```

Comment:
Mostly a note to myself: this method should theoretically be the same as the `DerefMut` implementation below, but actually behaves slightly differently. I want to look into why, and maybe even eliminate the duplication